po/id.po

```diff
@@ -8,14 +8,14 @@ msgstr ""
 "Project-Id-Version: flatpak master\n"
 "Report-Msgid-Bugs-To: https://github.com/flatpak/flatpak/issues\n"
 "POT-Creation-Date: 2022-03-14 15:32+0000\n"
-"PO-Revision-Date: 2022-01-26 08:44+0700\n"
+"PO-Revision-Date: 2022-03-28 20:35+0700\n"
 "Last-Translator: Kukuh Syafaat <kukuhsyafaat@gnome.org>\n"
```

Comment:
@rmnscnce next time, please update the last translator also

Reply:
> @rmnscnce next time, please update the last translator also

I see, haven't been aware of that. Will update it next time around.